.gitignore

```diff
@@ -120,6 +120,7 @@ ENV/
 /vfroot/
 /*.pid
 /scratches/
+logs
```

Comment:
please remove this part!

Reply:
This is an intended update. Each service saves logs under `./logs/`, and I think the directory should be excluded from the commit.